calicoctl/calico_ctl/node.py

```diff
@@ -277,7 +277,8 @@ def node_start(node_image, runtime, log_dir, ip, ip6, as_num, detach,
     # Get IP address of host, if none was specified
     if not ip:
         ips = get_host_ips(exclude=["^docker.*", "^cbr.*",
-                                    "virbr.*", "lxcbr.*"])
+                                    "virbr.*", "lxcbr.*", "veth.*",
```

Comment:
Should this include tunl0? (since we know we sometimes set up an interface called tunl0 with an IP address on it?)